manifests/server.pp

```diff
@@ -644,46 +644,46 @@
 		#ACTION      SOURCE DEST                PROTO DEST  SOURCE  ORIGINAL
 		#                                             PORT  PORT(S) DEST
 		shorewall::rule { 'http': rule => "
-		HTTP/ACCEPT  ${net}    $FW
+		HTTP/ACCEPT  ${net}    $fw
```

Comment:
nitpick, technically we could use ${fw} everywhere now to make it clearer. Optional.